plugins/services/src/js/utils/__tests__/TaskTableUtil-test.js

```diff
@@ -52,7 +54,7 @@ describe('TaskTableUtil', function () {
       }
     });
 
-    this.sortFunction = TaskTableUtil.getSortFunction('name');
+    this.getComparator = TaskTableUtil.getSortFunction('name');
```

Comment:
If we decide to call the sort functions now comparator, which I do like, we should do this in a separate PR and apply it to the complete code base.